conbench/app/benchmarks.py

```diff
@@ -226,7 +265,9 @@ def show_benchmark_results(bname: str, caseid: str) -> str:
 
     # Need to find a way to put bytes straight into jinja template.
     # still is still a tiny bit faster than using stdlib json.dumps()
-    infos_for_uplots_json = orjson.dumps(infos_for_uplots).decode("utf-8")
+    infos_for_uplots_json = orjson.dumps(
+        infos_for_uplots, option=orjson.OPT_INDENT_2
+    ).decode("utf-8")
```

Comment:
With the changes in this PR, the JSON doc is now a little easier to look at / debug based on the HTML source alone:
![Screenshot from 2023-05-06 17-00-43](https://user-images.githubusercontent.com/265630/236856598-64b6b1f1-5228-41b5-b5d9-0109c255cd83.png)

(of course, browser debug tools also help, but I am not a big fan of having an endless stream of numbers on a single line when doing dev work, and trying to keep an overview of what's where).